conkit/core/contactmap.py

```diff
@@ -394,6 +394,29 @@ def singletons(self):
                 singletons.remove(contact.id)
         return singletons
 
+    @property
+    def highest_contact(self):
```

Comment:
no, please don't. This name means nothing, highest contact tells me nothing about what this property contains.

what about highest_residue_number and you return the max (which seems to be what you want). alternatively, return "last_contact_by_resid" on the basis of residue 1 + residue 2 index.